lua/plenary/log.lua

```diff
@@ -129,7 +130,12 @@ log.new = function(config, standalone)
         if config.highlights and level_config.hl then
           vim.cmd "echohl NONE"
         end
-      end)
+      end
+      if config.use_console == 'sync' then
```

Comment:
we should check that we're not in a fast event here, otherwise this will error (which is not what you want from the logging)

Reply:
added in_fast_event check